slirp4netns.c

```diff
@@ -253,7 +253,7 @@ Slirp *create_slirp(void *opaque, struct slirp4netns_config *s4nn)
     Slirp *slirp = NULL;
     SlirpConfig cfg;
     memset(&cfg, 0, sizeof(cfg));
-    cfg.version = 1;
+    cfg.version = 3;
```

Comment:
This should be set to 3 only if v3 features are used

Reply:
done